docs/integrations/builtin/core-nodes/n8n-nodes-base.datatable/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Data table
+description: Documentation for the data table node in n8n, a workflow automation platform. Includes guidance on usage, and links to examples.
+contentType: [integration, reference]
+priority: critical
+tags:
+  - data table node
+  - data
+hide:
+  - tags
+search:
+  boost: 1.5
+---
+
+# Data table
+
+The Data Table node allows you to permanently save data across workflow executions in a table format. It provides functionality to perform various data operations on stored data.
+
+## Node parameters
+
+### Resource
+
+Select the resource on which you want to operate
+
+- Rows
+
+### Operations
+
+Select the operation you want to run on the resource:
+
+| Operation | Description | Options |
```

Comment:
Would it make sense to order these rows alphabetically to match the UI ordering?

Reply:
Also, is it worth mentioning the "Mapping column mode" (for update, insert, and upsert) and the "Conditions" selection (for delete, get, update, and upsert)?